### PR TITLE
openvpn 2.4.4

### PR DESCRIPTION
**Pull request by DomT4**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

Keychain support was removed again in:
https://github.com/OpenVPN/openvpn/commit/59e7e9fce8de6ea90d13baeaede83adc0b594e22

### CODE REVIEW

File: Formula/openvpn.rb [hunk 1]
```diff
@@ -63,7 +55,7 @@ def install
     (etc/"openvpn").install doc/"samples/sample-config-files/server.conf"
 
     # We don't use PolarSSL, so this file is unnecessary & somewhat confusing.
```
Comment (ilovezfs) [inline Formula/openvpn.rb]: Does this comment need updating too?

Comment (ilovezfs) [reply Formula/openvpn.rb]: :bell:

Comment (ilovezfs) [reply Formula/openvpn.rb]: :bellhop_bell:

Comment (DomT4) [reply Formula/openvpn.rb]: Sorry, had to sleep/study/life 😉. Yes, this comment can be renamed. Will do now.

Comment (ilovezfs) [reply Formula/openvpn.rb]: hehe I'm just messing with you as usual

Comment (DomT4) [reply Formula/openvpn.rb]: If I was feeling cruel I might've just left this for a few days to see whether you'd edit it yourself, but, security fixes and all that 😈.

**Comment by ilovezfs** [comment]
:open_umbrella: :v: :pen:

**Comment by DomT4** [comment]
Ta ❤️

**Pull request closed without merging**